transformation-scripts/example-template/example-geoglam-ingest.ipynb

```diff
@@ -76,10 +76,15 @@
    "source": [
```

Comment:
Should this notebook be moved to veda-docs?

Reply:
Agree that this should move to veda-docs